src/App.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './App.scss';
+import Person from './components/Person';
+
+// eslint-disable-next-line
+const misha = {
+  name: 'Misha',
+  age: 37,
+  sex: 'm',
+  isMarried: true,
+  partnerName: 'Natasha',
+};
+
+// eslint-disable-next-line
+const olya = {
+  name: 'Olya',
+  sex: 'f',
+  isMarried: true,
+  partnerName: 'Maksym',
+};
+
+// eslint-disable-next-line
+const alex = {
+  name: 'Alex',
+  age: 25,
+  sex: 'm',
+  isMarried: false,
+};
+
+const App = () => (
+  <div className="App">
+    <Person
+      name={misha.name}
+      age={misha.age}
+      sex={misha.sex}
+      isMarried={misha.isMarried}
+      partnerName={misha.partnerName}
+    />
+    <Person
+      name={olya.name}
+      sex={olya.sex}
+      isMarried={olya.isMarried}
+      partnerName={olya.partnerName}
+    />
+    <Person
+      name={alex.name}
+      age={alex.age}
+      sex={alex.sex}
+      isMarried={alex.isMarried}
+    />
```

Comment:
здається немає потреби розписувати всі пропси так детально:
```suggestion
    <Person {...misha} />

    <Person {...olya} />

    <Person {...alex} />
```

Reply:
та не потрібно було App.js змінювати на App.jsx

Reply:
краще не використовувати спреди зовсім, тому що тобі ми не контролюємо що попадає в пропси, но в цій задачі, це поки що ок)